tests/features/steps/2_request.py

```diff
@@ -82,9 +85,53 @@ def create_post_document_reference_step(context: Context, ods_code: str):
 
     if context.response.status_code == 201:
         doc_ref_id = context.response.headers["Location"].split("/")[-1]
+        doc_ref_id.replace(
+            "|", "."
+        )  # NRL-766 define and resolve custodian suffix behaviour
         context.add_cleanup(lambda: context.repository.delete_by_id(doc_ref_id))
 
 
+@when(
+    "producer 'TSTCUS' requests creation of a DocumentReference with default test values except '{section}' is"
```

Comment:
It can be, but I felt it would cause more confusion, beecause I wanted a helper method that would allow us to change one component to be non-compliant where _everything else_ is perfectly valid (so that we would know the component causing the failure would be the one we alter). 
The custodian changing would mean multiple components would become invalid for business logic reasons, since the producer ODS code sending the pointer must match the ODS code in multiple places in the body: custodian component, pointer ID and (if superceding) the context.relatesTo.